uint32 command
uint32 mClientId
uint32 UID
uint32 mnId
float64[3] mWorldPos

uint32 nNextId
int32 mnFirstKFid
int32 mnFirstFrame
int32 nObs

float32 mTrackProjX
float32 mTrackProjY
float32 mTrackProjXR
bool mbTrackInView
int32 mnTrackScaleLevel
float32 mTrackViewCos
uint64 mnTrackReferenceForFrame
uint64 mnFuseCandidateForKF

int32 mnVisible
int32 mnFound
float32 mfMinDistance
float32 mfMaxDistance

Observation[] mObservations
uint8[32] mDescriptor